autom8s/helm.js

```diff
@@ -48,18 +52,30 @@ class Helm {
 
   async delete(delOptions) {
     const { releaseName } = delOptions;
-    console.log(`deleting release: ${releaseName}`);
+    Helm._verifyNotEmpty(releaseName, 'releaseName');
 
+    console.log(`deleting release: ${releaseName}`);
     return this._executeHelm(`delete ${releaseName}`);
   }
 
   async upgrade(deployOptions) {
     const chartName = deployOptions.chartName.toLowerCase();
     const releaseName = deployOptions.releaseName.toLowerCase();
+
+    Helm._verifyNotEmpty(chartName, 'chartName');
```

Comment:
better rename to _validateNotEmpty or _assertNotEmpty - which is more conventional